libraries/chain/db_market.cpp

```diff
@@ -1003,6 +1003,10 @@ bool database::check_call_orders( const asset_object& mia, bool enable_black_swa
     if( !mia.is_market_issued() ) return false;
 
     const asset_bitasset_data_object& bitasset = ( bitasset_ptr ? *bitasset_ptr : mia.bitasset_data(*this) );
+    
+    // previously, price feeds could cause black swan in prediction market
+    if ( maint_time >= HARDFORK_CORE_460_TIME && bitasset.is_prediction_market )
```

Comment:
Please add a `// TODO: remove after HF` comment

Reply:
Then please add a log message for confirming this, and after confirming add a comment that it can't be removed. :-)